ci/run_travis_commands.sh

```diff
@@ -29,7 +29,16 @@ if [[ ${CMAKE_BUILD_TYPE} == Debug ]]; then
       cd ../regression_splitting && ./bin/run-tests.sh ../build/opensmt-splitter;
     fi
     cd ../regression_pipe && ./run-tests.sh ../build/opensmt;
-    cd ../regression_models && ./bin/run-tests.sh ../build/opensmt;
+    cd ../regression_models
+    if [ -f ./data/ENV-MODERN ]; then
+        ./bin/run-tests-dolmen.sh ../build/opensmt;
+    fi
+    if [ -f ./data/ENV-SMTCOMP ]; then
+        ./bin/run-tests-smtcomp.sh ../build/opensmt;
+    fi
+    if [ ! -f ./data/ENV-SMTCOMP -a ! -f ./data/ENV-MODERN ]; then
+        echo "Warning: the model regression environment is not set.  Skipping tests"
```

Comment:
Is this a valid situation? I am just wandering if this should not be an error which the script should propagate with `exit 1`...